src/main/kotlin/io/lsdconsulting/lsd/distributed/postgres/repository/InterceptedDocumentPostgresRepository.kt

```diff
@@ -11,8 +11,8 @@ import org.postgresql.util.PSQLException
 import javax.sql.DataSource
 
 
-private const val QUERY_BY_TRACE_IDS_SORTED_BY_CREATED_AT =
-    "select * from intercepted_interactions o where o.trace_id = ANY (?) order by o.created_at"
+private const val QUERY_BY_TRACE_IDS_LIMIT_100 =
+    "select * from intercepted_interactions o where o.trace_id = ANY (?) limit 100"
```

Comment:
A couple of suggestions:

- I think this value should be read from a property
- it would be nice if we had a test covering this value
- I may be wrong but removing the ordering from the query makes the result non-deterministic. If that's the case, is this what you'd expect? I think I'd expect the beginning of the flow to be displayed in the LSD (n oldest interactions) rather than ordered (in line 162) albeit random entries. What do you think?